Automattic/wp-calypso
# Media: Warn about Personal File Uploads in Guided Tours

#### Changes proposed in this Pull Request

In #38974, it was requested that the Media section inform users about file uploads being public whilst avoiding cluttering the interface. I think Guided Tours is the best way to do this, since that way, new users quickly get the information without seeing it again. 

#### Testing instructions

Visit `/media/site?tour=mediaBasicsTour` and verify that this appears.

<img width="1268" alt="Screenshot 2020-01-24 at 21 58 51" src="https://user-images.githubusercontent.com/43215253/73107015-bc480400-3ef4-11ea-84d1-ee295aff6cc0.png">

I suspect this might need a copy review? (cc @michelleweber) The copy is here:

> When your site is set to public visibility, all media files can be accessed by anyone, even if you have not included them in a post.
>
> It is therefore recommended that you do not upload any personal files.

Fixes #38974
